www/about/policies/terms-of-service.md

```diff
@@ -0,0 +1,414 @@
+Universal Public Domain Dedication</a>.
+
+  1. All content accessible through the Service is the sole responsibility of
+     the person who provides it, and Gratipay is not responsible for content
+provided by any Participant or third party. We do not represent or warrant the
+truthfulness, reliability, or accuracy of any content, or that the content does
+not infringe the rights of third parties. Gratipay shall not be responsible for
+any loss or damage arising out of your access to, use of, or reliance on the
+content.
+
+  1. Gratipay reserves the right to remove content for any reason, including
+     that it violates our Acceptable Use Policy. The Service may allow
+Participants to flag or report graphic, abusive, or otherwise objectionable
+content, and Gratipay may disable or filter access to flagged or reported
+content. However, Gratipay has no obligation to monitor, filter, or disable
+access to any content, and shall not be responsible if you encounter
+objectionable content by using the Service.
+
+  1. The Service, including Team profiles, may contain links to other websites.
+     When you access third-party websites linked to from the Service, you do so
+at your own risk. Gratipay does not review, control, or endorse those sites.
+
+1. License to Your Content
+
+  1. You retain your rights to any content you provide through the Service.
+     When you provide content through the Service, you grant to Gratipay a
+non-exclusive, worldwide, royalty-free, perpetual, license (including the right
+to sublicense) to use, host, store, cache, reproduce, publish, display,
+perform, distribute, transmit, adapt, and modify the content, in all media now
+known or later developed. 
+
+  1. This license permits Gratipay to operate, promote, and improve the
+     Service, and to develop new Service. It also permits Gratipay to provide,
+publish, or syndicate the content to third parties, for example by providing
+for the embedding of content on other services.
+
+  1. You represent and warrant that you have all necessary rights and authority
+     to grant this license to Gratipay. With regard to any third-party content
+that you submit, you represent and warrant that you have a license to provide
+the content according to these Terms, or that your use is otherwise
+permissible, for example as fair use.
+
+1. Acceptable Use Policy
+
+  1. The following conduct is prohibited on the Service:
+
+    * Illegal activities, including the promotion or solicitation of illegal
+      acts.
+
+    * Malicious activity or content, such as the transmission of viruses,
+      malware, or any other malicious or destructive code, or content or
+activity intended to harm or disrupt the hardware, networks, or infrastructure
+of Gratipay or any third parties.
+
+    * Unauthorized access, including: accessing or attempting to access any
+      non-public interface, system, or functionality of the Service, or testing
+the Service for vulnerabilities.
```

Comment:
We have a security disclosure policy that specifies the conditions under which we're okay with people "testing the Service for vulnerabilities," so how about this edit:

> testing the Service for vulnerabilities, except as described in our [security disclosure policy](https://gratipay.com/about/security/).